opensearch-3.yaml

```
@@ -0,0 +1,271 @@
+
+vars:
+  plugin_version_qualifier: "0" # external plugins use the same version as the main package with a qualifier
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - gradle
+      - openjdk-21-default-jdk
+      - patch
+      - unzip
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-21-openjdk
+    LANG: "en_US.UTF-8"
+
+data:
+  - name: internal-plugins
+    items:
+      # plugin_name: cve patch list
+      analysis-icu: ""
+      analysis-kuromoji: ""
+      analysis-nori: ""
+      analysis-phonetic: ""
+      analysis-smartcn: ""
+      analysis-stempel: ""
+      analysis-ukrainian: ""
+      crypto-kms: ""
+      discovery-azure-classic: ""
+      discovery-ec2: ""
+      discovery-gce: ""
+      identity-shiro: ""
+      ingest-attachment: ""
+      mapper-annotated-text: ""
+      mapper-murmur3: ""
+      mapper-size: ""
+      repository-azure: ""
+      repository-gcs: ""
+      repository-s3: ""
+      store-smb: ""
+      telemetry-otel: ""
+
+  - name: external-plugins
+    items:
+      # plugin_name: cve patch list
+      alerting: ""
+      anomaly-detection: ""
+      asynchronous-search: ""
+      cross-cluster-replication: ""
+      custom-codecs: ""
+      geospatial: ""
+      index-management: ""
+      job-scheduler: ""
+      k-nn: ""
+      ml-commons: ""
+      neural-search: ""
+      notifications: ""
+      observability: ""
+      performance-analyzer: ""
+      reporting: ""
+      security: ""
+      security-analytics: ""
+      sql: ""
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/opensearch-project/OpenSearch
+      tag: ${{package.version}}
+      expected-commit: dc4efa821904cc2d7ea7ef61c0f577d3fc0d8be9
+
+  - runs: |
+      echo "org.gradle.daemon=false" >> gradle.properties
+      gradle localDistro --parallel -Dbuild.snapshot="false" -Dbuild.version_qualifier=""
+      mkdir -p ${{targets.destdir}}/usr/share/opensearch/logs
+
+      install -dm777 ${{targets.destdir}}/usr/share/opensearch/
+      mv /home/build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/bin ${{targets.destdir}}/usr/share/opensearch
+      mv /home/build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/lib ${{targets.destdir}}/usr/share/opensearch
+      mv /home/build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/config ${{targets.destdir}}/usr/share/opensearch
+      mv /home/build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/modules ${{targets.destdir}}/usr/share/opensearch
+      mv /home/build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/plugins ${{targets.destdir}}/usr/share/opensearch
+      mv /home/build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/agent ${{targets.destdir}}/usr/share/opensearch
```

```suggestion
      mv build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/bin ${{targets.destdir}}/usr/share/opensearch
      mv build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/lib ${{targets.destdir}}/usr/share/opensearch
      mv build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/config ${{targets.destdir}}/usr/share/opensearch
      mv build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/modules ${{targets.destdir}}/usr/share/opensearch
      mv build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/plugins ${{targets.destdir}}/usr/share/opensearch
      mv build/distribution/archives/linux-*-tar/build/install/opensearch-${{package.version}}/agent ${{targets.destdir}}/usr/share/opensearch
```

Absolute dir works locally, and relative doesn't, maybe CI is opposite?